nixos/modules/services/video/wivrn.nix

```diff
@@ -0,0 +1,122 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.wivrn;
+in
+{
+  options = {
+    services.wivrn = {
+      enable = mkEnableOption (lib.mdDoc "WiVRn, an OpenXR streaming application.");
+
+      package = mkPackageOption pkgs "wivrn" { };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc ''
+          Whether to open the default ports in the firewall for the WiVRn server.
+        '';
+      };
+
+      defaultRuntime = mkOption {
+        type = types.bool;
+        description = ''
+          Whether to enable WiVRn Monado as the default OpenXR runtime on the system.
+
+          Note that applications can bypass this option by setting an active
+          runtime in a writable XDG_CONFIG_DIRS location like `~/.config`.
+        '';
+        default = false;
+        example = true;
+      };
+
+      highPriority = mkEnableOption "high priority capability for wivrn-server"
+        // mkOption { default = true; };
```

Comment:
I am not quite sure if WiVRn needs this. Monado uses this for asynchronous reprojection, but I *think* this would be handled by the HMD here

Reply:
It complained about not being able to set caps before I added that